test/cases/compile_errors/refer_to_the_type_of_a_generic_function.zig

```diff
@@ -8,4 +8,4 @@ export fn entry() void {
 // backend=stage2
 // target=native
 //
-// :4:5: error: use of undefined value here causes undefined behavior
+// :4:6: error: use of undefined value here causes undefined behavior
```

Comment:
@Vexu 

This regressed the test coverage that this test was covering. Now it's hitting the new error, instead of the old one. Losing test coverage is a merge blocker, unless there is a good reason to make an exception.